docs/xxcheckpointgcthread.md

```diff
@@ -0,0 +1,51 @@
+<!--
+* Copyright (c) 2017, 2023 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:CheckpointGCThreads
+
+**(Linux&reg; x86, Linux on POWER LE&reg;, and Linux on IBM Z&reg; only)**
+
+This option restricts the number of GC threads used at the time of taking checkpoint.
+
+:fontawesome-solid-triangle-exclamation:{: .warn aria-hidden="true"} **Restrictions:** This option takes effect only when `-XX:+EnableCRIUSupport` is enabled.
+
+## Syntax
+
+        -XX:CheckpointGCThreads=<Number_of_GC_threads>
+
+| Setting               | Value                            | Default                                                  |
+|-----------------------|-----------------------------------|:---------------------------------------------------------:|
+| `<Number_of_GC_threads>` | Greater than 0 and less than or equal to GC threads at VM startup |  4 GC threads            |
+
+You can specify the number of GC threads used at the startup with the `-xgcthreads` option. If a fixed GC thread count is not specified with the `xgcthreads` option, the VM uses the default GC thread count that is set to `n-1`, where `n` is the number of reported CPUs. For more information about the setting of the GC thread count, see the [`-Xgcthreads`](xgcthreads.md) topic.
+
+## Explanation
+
+When taking a checkpoint, the VM reduces the current number of GC threads that the VM started with to the number of GC threads specified for the checkpoint with the `-XX:CheckpointGCThreads` option. Reducing the GC thread count minimizes the restore time as restoring threads adds latency to the overall VM restore time. After restoring from the checkpoint, the VM will increase the current GC thread count back to the number of GC threads that VM had at the startup.
```

Comment:
"the current number of GC threads that the VM started with" doesn't make much sense.
Please see Word Usage for "as".
Please correct use of future tense.
Missing "the" in front of "VM"